modules/ppcp-settings/src/SettingsModule.php

```diff
@@ -203,6 +203,36 @@ static function () use ( $container ) : void {
 			}
 		);
 
+		add_action(
+			'admin_init',
+			function() use ( $container ) {
+				global $wp_filter;
+
+				if ( ! $container->has( 'wcgateway.is-ppcp-settings-page' ) ||
+					$container->get( 'wcgateway.is-ppcp-settings-page' ) === false ) {
+					return;
+				}
+
+				if ( empty( $wp_filter['woocommerce_sections_checkout']->callbacks ) ) {
+					return;
+				}
```

Comment:
Why not having like this?

```suggestion
				if ( 
					! $container->has( 'wcgateway.is-ppcp-settings-page' )
					|| ! $container->get( 'wcgateway.is-ppcp-settings-page' )
					|| empty( $wp_filter['woocommerce_sections_checkout']->callbacks )
				) {
					return;
				}
```

Reply:
@Narek13  I've moved the declaration of `$wp_filter` after the first check as suggested. Thanks 🙂